src/Commands/WaContent.php

```diff
@@ -125,6 +128,82 @@ private function createPost($waContent)
     private function handleTranslation($postId, $waContent)
     {
         LanguageProvider::setPostLanguage($postId, $waContent->translation->locale ?? 'da');
+
+        //if this is not the master translation, just return
+        if (!$waContent->translation->is_master || !isset($waContent->translation)) {
+            return;
+        }
+
+        $postTranslations = collect($waContent->translation->translation_ids);
+
+        if (empty($postTranslations)) {
+            WP_CLI::warning('found no translations. returning.');
+            return;
+        }
+
+        WP_CLI::success(sprintf('found the following translations: %s', json_encode($postTranslations, JSON_PRETTY_PRINT)));
+        $translationPostIds = $postTranslations->map(
+            function ($translation_id, $locale) use ($waContent) {
+                if (null == (env($endpoint = strtoupper('WHITEALBUM_ENDPOINT_'.$locale)))) {
+                    WP_CLI::warning(sprintf('%s has not been defined in your ENV file.', $endpoint));
+                    return;
+                }
+                $repository = new ContentRepository(env($endpoint));
+                $translation_post_id = null;
+                if ($translation_post_id = WpComposite::id_from_white_album_id($translation_id)) {
+                    WP_CLI::success(sprintf('found imported translation: %s', $translation_post_id));
+                } else {
+                    //if associated post isn't imported yet, check both articles and galleries
+                    $translation = collect(
+                        [
+                        'article' => ContentRepository::ARTICLE_RESOURCE,
+                        'gallery' => ContentRepository::GALLERY_RESOURCE,
+                        ]
+                    )->map(
+                        function ($endpoint, $contentType) use ($translation_id,$repository, $locale) {
```

Comment:
By doing this, you will always make two requests to WA, instead of only making one request, when the WA id is not an article, which most of them will be.
It would be better to always request for `ContentRepository::ARTICLE_RESOURCE` and then only request for the galleries, when there wasn't a match on the article request.